service/token/service.go

```diff
@@ -89,6 +109,27 @@ func (s *service) Create(config CreateConfig) ([]string, error) {
 	return tokens, nil
 }
 
+func (s *service) CreateOrgPolicy(clusterID string) error {
+	// Get the system backend for policy operations.
+	sysBackend := s.VaultClient.Sys()
+
+	// Create organization policy name and HCL policy rules.
+	orgPolicyName := s.OrgPolicyName(clusterID)
+	organizationsRoleHash := computeRoleHash(systemMastersOrganizations)
+	rules, err := execTemplate(pkiIssueOrgPolicyTemplate, pkiIssueOrgPolicyContext{ClusterID: clusterID, OrganizationsRoleHash: organizationsRoleHash})
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	// Actually create the policy within Vault.
+	err = sysBackend.PutPolicy(orgPolicyName, rules)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
```

Comment:
Doesn't it make sense to extract common method `createOrgPolicy(clusterID, name rules)` (and something similar for deletion)?

Reply:
I talked about this with @xh3b4sd . The overall approach doesn't fix `certctl issue` command with the configured organization. That's gonna need a redesign of application. Currently, I'm just trying to fix the issue we have. I know it is a dumb way, but any other requires a different amount of work.

Reply:
Is it temporary? Do we have issue tracking fixing this?

Reply:
I don't if it is temporary. @xh3b4sd said this needs to be rewritten at all. I don't know about this, but in current case I don't see any easy way to fix this. Details here https://github.com/giantswarm/hive/pull/403/files#diff-dad3192ce74750d74aeefb916c26bb6bR19

We need either precreate that org role for `system-masters` or allow nodes to have tokens with wider access. As we now we don't have any case, where we use `organizations` other, that for `system:masters` - I'd like to go with this and solve the issue in a wider context when this issue appears. If no - I'll close that and we can stick with ansible playbook fix, which is even worse.

Reply:
OK. At this point I don't have opinion on that. Don't feel blocked by me.